pom.client.xml

```diff
@@ -1231,6 +1231,7 @@
     <module>sdk/core/azure-core-test</module>
     <module>sdk/core/azure-core-tracing-opentelemetry</module>
     <module>sdk/e2e</module>
+    <module>sdk/servicebus/azure-messaging-servicebus</module>
```

Comment:
Did you also add this to the jacoco test coverage? I'm not sure if spotbugs aggregate report also requires an entry.

Reply:
@conniey, good catch. Looking at the spotbugs aggregate report pom file, it only seems to include data libraries. I think this just need an entry in jacoco.